solidity/contracts/isms/native/ArbitrumISM.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
consider making this folder name more descriptive
eg nativerollup or something

Reply:
and probably mirroring the folder structure in the hooks dir

Reply:
what will polygon go under?